app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1272,18 +1518,26 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope, DaxDialogLi
                 lastSeenOmnibarViewState = viewState
 
                 if (viewState.isEditing) {
-                    omniBarContainer.background = null
                     cancelAllAnimations()
-                } else {
-                    omniBarContainer.setBackgroundResource(R.drawable.omnibar_field_background)
                 }
 
                 if (shouldUpdateOmnibarTextInput(viewState, viewState.omnibarText)) {
                     omnibarTextInput.setText(viewState.omnibarText)
                     appBarLayout.setExpanded(true, true)
+                    decorator.updateBottomBarVisibility(true, true)
                     if (viewState.shouldMoveCaretToEnd) {
                         omnibarTextInput.setSelection(viewState.omnibarText.length)
                     }
+                } else {
+                    decorator.updateBottomBarVisibility(!viewState.isEditing)
+                }
+
+                if (ctaContainer.isVisible) {
```

Comment:
When the Widget CTA appears we want to make sure it's on top of the Bottom Navigation Bar. For another view (like we do with `Snackbar`) we could use the Behaviour, but because the Widget CTA is a `ConstraintLayout` we can't identify it the same way. Once the experiment is completed we will refactor if necessary